pkg/scheduler/algorithm/predicates/csi_volume_predicate_test.go

```diff
@@ -452,7 +535,8 @@ func TestCSIVolumeCountPredicate(t *testing.T) {
 				expectedFailureReasons = []PredicateFailureReason{test.expectedFailureReason}
 			}
 
-			pred := NewCSIMaxVolumeLimitPredicate(getFakeCSIPVInfo(test.filterName, test.driverNames...),
+			pred := NewCSIMaxVolumeLimitPredicate(getFakeCSIDriverInfo(test.ignoreCSIDriver, test.driverNames...),
```

Comment:
Should probably rename this predicate because now it does more than only checking the volume limits.

Reply:
predicates become part of API and an user may refer this predicate in scheduler.json file. Renaming the predicate will has to follow a deprecation cycle and stuff.